draft-ietf-ppm-dap.md

```diff
@@ -1499,6 +1518,45 @@ its query:
 * For fixed-size tasks, the batch selector is the batch ID assigned sent in the
   response.
 
+### Collect Message Security
+
+Collect sub-protocol messages must be confidential and mutually authenticated.
+
+HTTPS provides confidentiality and authenticates the leader to the collector.
+Additionally, the leader encrypts its aggregate share to a public key held by
+the collector.
+
+Collectors MUST authenticate their requests to leaders using a scheme that meets
+the requirements in {{request-authentication}}.
+
+[[OPEN ISSUE: collector public key is currently in the task parameters, but this
+will have to change #102]]
+
+The collector and helper never directly communicate with each other, but the
+helper does transmit an aggregate share to the collector through the leader, as
+detailed in {{collect-aggregate}}. The aggregate share must be confidential from
+everyone but the helper and the collector.
+
+Confidentiality is achieved by having the helper encrypt its aggregate share to
+a public key held by the collector.
+
+There is no authentication between the collector and the helper. This allows the
+leader to:
+
+* Send collect parameters to the helper that do not reflect the parameters
+  chosen by the collector
+* Discard the aggregate share computed by the helper and then fabricate
+  aggregate shares that combine into an arbitrary aggregate result
+
+These are both attacks on soundness, which we already assume to hold only if
+both aggregators are honest, which puts these malicious-leader attacks out of
+scope.
+
+[[OPEN ISSUE: Should we have authentication in either direction between the
+helper and the collector? #155]]
+[[OPEN ISSUE: DAP and/or VDAF should spell out what "soundness" and "privacy"
+mean]]
```

Comment:
VDAF does already. Does DAP not as well?

Reply:
Oh, whoops, I was searching for "soundness" in VDAF, but it discusses "robustness". And in security considerations we discuss "privacy" and "correctness", but the doc has references to "soundness" and "robustness", both in protocol text and the odd OPEN ISSUE block. I filed #306 to clean up this language in DAP and align it with VDAF.